srcs/layouts/shaw_imperial_en.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Shaw Imperial layout for shavian alphabet, see https://www.shavian.info/keyboards/ -->
+<keyboard name="Shaw Imperial" script="shavian">
+  <row>
+    <key key0="𐑶" key2="1" key4="esc"/>
+    <key key0="𐑬" key1="~" key2="2" key3="\@"/>
+    <key key0="𐑫" key1="!" key2="3" key3="\#"/>
+    <key key0="𐑜" key2="4" key3="$"/>
+    <key key0="𐑖" key1="𐑠" key2="5" key3="%"/>
+    <key key0="𐑗" key2="6" key3="^"/>
+    <key key0="𐑙" key2="7" key3="&amp;"/>
+    <key key0="𐑘" key2="8" key3="*"/>
+    <key key0="𐑡" key2="9" key3="(" key4=")"/>
+    <key key0="𐑔" key2="0" key3="f11_placeholder" key4="f12_placeholder"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="𐑭" key1="tab" key2="𐑸" key3="`"/>
```

Comment:
The alternating shift avoids having a grid layout but make the keys smaller. What do you think of a grid layout ?

Reply:
I thought about that, yes. I would argue that it makes it a bit different, but maybe compactness matters more here, I'm fine with making it a grid.